app/controllers/world_locations_controller.rb

```diff
@@ -28,6 +30,23 @@ def show
         set_meta_description("What the UK government is doing in #{@world_location.name}.")
         set_slimmer_world_locations_header([@world_location])
         set_slimmer_organisations_header(@world_location.worldwide_organisations_with_sponsoring_organisations)
+        # Display the "B" variant of this page for users in the "B" bucket
+        ab_test = GovukAbTesting::AbTest.new("WorldwidePublishingTaxonomy", dimension: 45)
+        @requested_variant = ab_test.requested_variant(request.headers)
+        @requested_variant.configure_response(response)
+        if @requested_variant.variant_b?
+          parent_taxon = {
```

Comment:
We could move `parent_taxon` and `accordion_content` into private methods? Might declutter this controller action a bit?